docs/architecture/deep-dives/passkeys/implementations/relying-party/prf.md

```diff
@@ -0,0 +1,101 @@
+# Passkeys for Decryption
+
+When a user logs into their Bitwarden vault, two related but distinct operations take place -
+authentication and decryption.
+
+When using a master password, the password itself serves both of these purposes - authenticating the
+user by verifying the password was entered correctly, and allowing decryption of your data through
+the derivation of an encryption key from the password. For more detail on this, see the
+[Bitwarden Security Whitepaper](https://bitwarden.com/help/bitwarden-security-white-paper/).
+
+Passkeys inherently provide the mechanism for authentication. The `prf` extension introduced in the
+WebAuthn [specification](https://w3c.github.io/webauthn/#prf-extension) provides the framework
+necessary for passkeys to be used for decryption as well.
+
+## What is PRF?
+
+At the most abstract level, account encryption and decryption requires that the user's symmetric key
+be encrypted ("protected") with another key that is available only to the client. For master
```

Comment:
question: do you think "client" might be confusing? I first thought of the WebAuthn Client, but in this context we are referring to the "client application"?

suggestions (non-blocking):
```suggestion
be encrypted ("protected") with another key that is available only to the client application. For master
```

```suggestion
be encrypted ("protected") with another key that is only available client-side. For master
```


```suggestion
be encrypted ("protected") with another key that is available only to the user. For master
```


```suggestion
be encrypted ("protected") with another key that is never shared with the server. For master
```

Reply:
I addressed this with [`4ab27ff` (#307)](https://github.com/bitwarden/contributing-docs/pull/307/commits/4ab27ff50f7a78620ea68d6f99691c481f24c263).